src/rust-2018/rustup-for-managing-rust-versions.md

```diff
@@ -209,4 +214,4 @@ $ rustup component add llvm-tools-preview
 ```
 
 This is the newest component, and so doesn't have good documentation at the
-moment.
\ No newline at end of file
+moment.
```

Comment:
There is no change on this line.  Any idea why it's showing that?

Reply:
You removed a space at the end of the line, which is good.

Reply:
Interesting.  Vim must have done that automatically with one of the various plugins I've installed.